.github/workflows/pull.yml

```diff
@@ -285,34 +285,6 @@ jobs:
       docker-image: ${{ needs.linux-focal-cuda12_1-py3_10-gcc9-build.outputs.docker-image }}
       test-matrix: ${{ needs.linux-focal-cuda12_1-py3_10-gcc9-build.outputs.test-matrix }}
 
-  linux-focal-cuda12_4-py3_10-gcc9-build:
-    name: linux-focal-cuda12.4-py3.10-gcc9
-    uses: ./.github/workflows/_linux-build-label.yml
-    with:
```

Comment:
What do you think about putting these in trunk instead of periodic, to serve as smoke tests? (while keeping the rest in periodic)

Reply:
Telling from current HUD read. CUDA 12.4 inductor_timm shard seems to be most helpful. 
https://hud.pytorch.org/hud/pytorch/pytorch/main/1?per_page=50&name_filter=inductor_timm 
Since CUDA 12.1 inductor_timm shard is mostly un-usable. Perhaps, just delete all the shards for cuda 12.4 except inductor_timm? 
And agree with @ZainRizvi trunk job runs might still be necessary to detect regressions.

Reply:
kept inductor_timm in inductor, kept all others in inductor-periodic
sm86 cu 124 -> trunk
cu124 -> periodic